backend/cmm_helpers.ml

```diff
@@ -2336,17 +2336,45 @@ let ptr_offset ptr offset dbg =
 let direct_apply lbl ty args (pos, _mode) dbg =
   Cop (Capply (ty, pos), Cconst_symbol (lbl, dbg) :: args, dbg)
 
+let split_two_halves ~len l =
```

Comment:
This name is confusing: it implies that the list is split into two equally sized halfs. The function is also already available as `Misc.Stdlib.List.split_at`